test/transpile.jl

```diff
@@ -73,19 +80,25 @@ end
     transpiler = CompressSingleQubitGatesTranspiler()
```

Comment:
We also have to test cases where we have two qubits and a readout, that it correctly compresses the two-qubits and the readout stays in-tact.

Reply:
added